packages/rest/package.json

```diff
@@ -36,10 +39,21 @@
 		"dist"
 	],
 	"dependencies": {
-		"tslib": "^2.0.1"
+		"@discordjs/collection": "^13.0.0-dev",
+		"@discordjs/core": "^13.0.0-dev",
+		"@discordjs/form-data": "^3.0.1",
+		"@sapphire/async-queue": "^1.0.2",
+		"@sapphire/snowflake": "^1.2.4",
+		"abort-controller": "^3.0.0",
+		"discord-api-types": "^0.12.1",
+		"node-fetch": "^2.6.1",
+		"tslib": "^2.1.0"
 	},
-	"devDependencies": {},
 	"engines": {
 		"node": ">=14.0.0"
+	},
+	"devDependencies": {
+		"@types/nock": "^11.1.0",
```

Comment:
Types is installed but not the actual `nock` package?

Reply:
Oops... Both nock and it's types need to be installed on root, not on this package. Will fix!